src/components/Products/ProductCardView/ProductCard.jsx

```diff
@@ -1,10 +1,15 @@
 import './productCard.css';
-import { useProduct } from "../../../contexts/index";
-import { addProductToCart, addProductToWishlist, deleteProductFromWishlist} from "../../../services/index";
-import { useAuth } from '../../../contexts/index'; 
-import { isItemInCart } from "../../../utils/cart/cart";
-import { isItemnWishlist } from "../../../utils/wishlist/wishlist";
 import { useNavigate } from 'react-router';
```

Comment:
It's good if you use `react-router-dom` instead of `react-router`. Do read about the differences

Reply:
Will read about it.